.github/workflows/auto_cherry_pick.yml

```diff
@@ -43,3 +43,37 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+      - name: Set issue title
+        if: ${{ failure() }}
+        run: |
+            echo "title=[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}" >> "$GITHUB_ENV"
+      - name: Create Issue on Failed workflow
+        if: ${{ failure() }}
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
+          title: |
+            ${{ env.title }}
+          body: |
+            #### Auto-Cherry-Pick WorkFlow Failure:
+
+            - [Failed Run](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+
+            - [PR Number](https://github.com/${{ github.repository }}/pull/${{ github.event.number }})
+
+            - Workflow Name - `${{ github.workflow }}`
+
+            - Job -           `${{ github.job }}`
+
+            - Status -        `${{ job.status }}`
```

Comment:
I think these are the least necessary! Feel free to remove as its auto understood thing.